src/components/InputTime/utils.test.tsx

```diff
@@ -118,7 +118,7 @@ describe('getEndOfDay', () => {
     expect(endOfDay.toISOString()).toEqual(expectedDateString.toISOString());
   });
 
-  describe('with optional `timeZoneOffset`', () => {
+  describe.skip('with optional `timeZoneOffset`', () => {
```

Comment:
Why are these tests being skipped?

Reply:
This is my bad.

TLDR: I've been working on timeZoneOffset for a while - it's now on the backburner and not fully functional.

I have a long-running WIP to add a timezone offset to the `InputTime` picker.
Ideally, it would let consumers do something like 
```
<InputTime
  min="2020-12-07T22:33:05.925Z" 
  timeZoneOffset={-330}
/>
```
So they'd get a dropdown list starting with the time of day for the min date, and ending at midnight in the chosen location. I wrote it without using a library, and it's been a nightmare.

I keep finding edge cases, for instance, now that daylight savings is over, the tests fail.

I told Boima to ignore it for now, I need to circle back and fix it or delete the code.